src/shogun/regression/svr/LibLinearRegression.cpp

```diff
@@ -38,6 +38,7 @@ void CLibLinearRegression::init_defaults()
 {
 	set_C(1.0);
 	set_epsilon(1e-2);
+	set_tube_epsilon(0);
```

Comment:
sklearn uses 0.1, but the test won't pass using this value (error > 1e-5)

Reply:
Ah nice! If you want you can adjust the parameters around a bit? Thanks anyway!